src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,523 @@
+                                    !validate(inputAddress) &&
+                                    classes.errorStyle
+                                }`,
+                                notchedOutline: classes.notchedOutlineStyles,
+                            },
+                        }}
+                        helperText={
+                            addressTouched &&
+                            !validate(inputAddress) && <InputErrorText />
+                        }
+                        FormHelperTextProps={{
+                            className: classes.helperText,
+                        }}
+                        error={addressTouched && !validate(inputAddress)}
+                    />
+                </div>
+                <div
+                    className={`${classes.inputSectionWrapStyles} ${classes.wrapStyles}`}
+                >
+                    <Typography component="h2" className={classes.titleStyles}>
+                        Country
+                    </Typography>
+                    <Typography
+                        component="h4"
+                        className={classes.subTitleStyles}
+                    >
+                        Select the country where the production site is located.
+                    </Typography>
+                    <StyledSelect
+                        id="country"
+                        name="Country"
+                        aria-label="Country"
+                        options={countriesOptions || []}
+                        value={inputCountry}
+                        onChange={handleCountryChange}
+                        className={classes.selectStyles}
+                        styles={selectStyles}
+                        placeholder="Country"
+                        isMulti={false}
+                    />
+                </div>
+                <hr className={classes.separator} />
+                <div
+                    className={`${classes.sectionWrapStyles} ${classes.wrapStyles}`}
+                >
+                    <div className={classes.rowContainerStyles}>
+                        <Typography
+                            component="h2"
+                            className={`${classes.titleStyles} ${classes.marginRight}`}
+                        >
+                            Additional information
+                        </Typography>
+                        <div
+                            className="cursor"
+                            onClick={toggleExpand}
+                            onKeyDown={toggleExpand}
+                            role="button"
+                            styling="link"
+                            tabIndex={0}
+                        >
+                            {isExpanded ? (
+                                <ArrowDropUpIcon />
+                            ) : (
+                                <ArrowDropDownIcon />
+                            )}
+                        </div>
```

Comment:
Can you try to use the MUI `IconButton` component as a wrapper? I think It is more suitable here.

Reply:
done